flac.md

```diff
@@ -363,13 +363,25 @@ Note that the Vorbis comment as used in Vorbis allows for on the order of 2\^64
 
 ### Standard field names
 
-Only one standard field name is defined; the channel mask field, in (#channel-mask). In general, most FLAC playback devices and software recognize the following field names:
+Only one standard field name is defined: the channel mask field, in (#channel-mask). No other field names are defined because the applicability of any field name is strongly tied to the content it is associated with. For example, field names useful for describing files that contain a single work of music would be unusable when labeling archived broadcasts, recordings of any kind, or a collection of music works. Even when describing a single work of music, different conventions exist depending on the kind of music: orchestral music differs from music by solo artists or bands.
+
+Despite the fact that no field names are formally defined, there is a general trend among devices and software capable of FLAC playback that are meant to play music. Most of those recognize at least the following field names:
```

Comment:
```suggestion
Despite the fact that no field names are formally defined, there is a general trend among devices and software capable of FLAC playback that are meant to play music. An implementation MAY recognize the following field names:
```

For defining the level of RFC 2119 recommendation (here: optional)

Reply:
As part of the shepherds review, a lot of RFC 2119 keywords were removed because they were in informative parts of the specification (like the section conceptual overview), and they don't really belong there. I think the same goes here: this sentence gives a hint on what a specific set of implementations often do, which isn't normative at all. So I think RFC 2119 keywords aren't really useful here?